e2e/cluster.go

```diff
@@ -18,6 +18,14 @@ const (
 	managedClusterResourceType = "Microsoft.ContainerService/managedClusters"
 )
 
+// This map is used during cluster creation to check what VM size should
+// be used to create the single default agentpool used for running cluster essentials
+// and the jumpbox pods/resources.
+var locationToDefaultClusterAgentPoolVMSize = map[string]string{
+	"eastus":         "Standard_DS2_v2",
+	"southcentralus": "Standard_ND96isr_H100_v5",
```

Comment:
We shouldn't default to this SKU for southcentralus because there's very limited capacity and it's expensive. 
Having a variable to update the SKU once and have it be handled everywhere would be nice though

Reply:
I guess we ned to figure out which default we _can_ us here, it seems like these expensive h100-based SKUs are the only ones to choose from there?

Reply:
We don't have quota for this but we'll have to get quota for a cheaper one.